### PR TITLE
Fixes invisible lings (Carbonhell)

### LINKED ISSUE #1918: Bug Report-Changling Perma Invisible
When playing as a changling, being chamelion skinned, then transforming into another said person you can walk around without uncloaking, attack, and basically do anything a person can do but basically being unseen. Works until said player toggles chamelion skin again which will uncloak them.

### PR DESCRIPTION
**Pull request by Kokojos**
Re-ports Carbonhell's fixes to invisible lings. All credits to him

Fixes #1918

### CODE REVIEW

**Comment by Nokington** [comment]
:+1:

**Comment by Nebyab** [comment]
why not

File: code/datums/mutations.dm [hunk 1]
```diff
@@ -253,7 +253,8 @@ var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
 	text_gain_indication = "<span class='danger'>You feel strange.</span>"
 
 /datum/mutation/human/bad_dna/on_acquiring(mob/living/carbon/human/owner)
-	owner << text_gain_indication
```
Comment (Carbonhell) [inline code/datums/mutations.dm:256]: i don't remember but are you sure this has to be removed

Comment (Kokojos) [inline code/datums/mutations.dm:256]: That's what you did.

**Pull request merged**